executor/builder.go

```diff
@@ -4995,8 +5002,14 @@ func (b *executorBuilder) buildWindow(v *plannercore.PhysicalWindow) exec.Execut
 			expectedCmpResult: cmpResult,
 		}
 
-		tmpProcessor.start.InitCompareCols(b.ctx, orderByCols)
-		tmpProcessor.end.InitCompareCols(b.ctx, orderByCols)
+		err = tmpProcessor.start.UpdateCompareCols(b.ctx, orderByCols)
```

Comment:
Ditto.

Reply:
> Ditto.

UpdateCompareCols is already called in buildWindowFunctionFrameBound, however, we need to call it again as the meta data in orderByCols may be changed.